docs/roadmap/README.md

```diff
@@ -0,0 +1,38 @@
+# React Cosmos v6
+
+## Bundler
+
+- [ ] Revamp webpack integration.
+  - [ ] Make webpack and external package (and plugin).
```

Comment:
I am literally in the middle of looking at this today in our monorepo and wondering how to deal with potentially divergent versions of Webpack when it is impacted by different tools. I love that we are considering making it external.

Reply:
Almost all Cosmos issues are webpack-related. Trying to stay compatible with all webpack versions (and all webpack config variations used by various frameworks) out of the box has already been crazy and it's only getting worse as time passes and more versions are released. Would be great to find a more scalable way to deal with webpack and bundler integration in general.